src/shapes/lsystem/subtypes.js

```diff
@@ -24,6 +24,51 @@ export const subtypes = {
     minIterations: 1,
     maxIterations: 8
   },
+  // https://onlinemathtools.com/l-system-generator
+  'Fractal Tree': {
+    axiom: 'F',
+    draw: ['F'],
+    rules:  {
+      F: 'F=F[+FF][-FF]F[-F][+F]F',
```

Comment:
Should this have the `F=`?

Reply:
No, and the tree looks way better when it's fixed. Do you want me to submit a PR?